src/Toast/index.jsx

```diff
@@ -7,14 +7,14 @@ import { Button, IconButton, Icon } from '..';
 import { Close } from '../../icons';
 
 function Toast({
-  action, children, closeLabel, delay, onClose, show,
+  action, children, className, closeLabel, delay, onClose, show,
 }) {
   const [autoHide, setAutoHide] = useState(true);
   return (
     <ToastContainer>
       <BaseToast
         autohide={autoHide}
-        className="toast"
+        className={className}
```

Comment:
As suggested in previous comment, I think we should include `pgn__toast` here, i.e. `className={classNames('pgn__toast', className)}`.

Similarly, we should also adopt the prop spreading strategy here that I previously mentioned as well, with regard to `{...rest}`:

```jsx
function Toast({
  className,
  ...rest
}) {
  return (
    <ToastContainer>
      <BaseToast
        className={classNames('pgn__toast', className)
        {...rest}
      />
    </ToastContainer>
  );
}
```